Peer.Domain/PullRequestStatus.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Peer.Domain
+{
+    public enum PullRequestStatus
+    {
+        Unknown,
+        Draft,
+        Conflict,
+        FailedChecks,
+        ActionsQueued,
+        ActionsPending,
```

Comment:
Yeah

Reply:
Like when github hasn't started running actions and it looking for a worker.. or in azdo when you're trying to schedule on a pool that's already full

Reply:
Okay when I first saw pending I took as “running is pending “ which seemed weird cause it would mean there wa s I status for running. Iirc there are 2 different pre-running statuses but I don’t think we care about the difference even if there is one